lib/client.js

```diff
@@ -216,8 +217,20 @@ exports = module.exports = class Tgfancy extends TelegramBot {
 
         // We are now performing 'on ready' actions.
         // Let's set the correct webhook when on Openshift
-        if (this.options._openshift) {
+        if (this.options._openshift && process.env.OPENSHIFT_APP_UUID) {
```

Comment:
Yeah, that's a good point, this PR is gonna be small, `git merge --squash` sounds like a good idea